src/screens/Departures/components/EstimatedCallItemWrapperView.tsx

```diff
@@ -0,0 +1,33 @@
+import {PlaceScreenMode} from '@atb/screens/Departures/PlaceScreen';
+import {TouchableOpacity, View} from 'react-native';
+import {StyleSheet} from '@atb/theme';
+import {ReactNode} from 'react';
+
+const EstimatedCallItemWrapperView = ({
```

Comment:
Could consider making this a more generic component, as I feel the naming `EstimatedCallItemWrapperView` is kinda hard to understand, and a developer have to read the code to get an idea what it actually does. 

Maybe something like:
```
const TouchableOrView = ({onClick, style, children}) => 
  onClick
    ? <TouchableOpacity onPress={onClick} style={style}>{children}</TouchableOpacity>
    : <View style={style}>{children}</View>
```

You don't necessarily have to do this now, we don't want PRs to drag out forever. You can decide based on how you feel about the suggestion 😄

Reply:
I did think about this, but then I couldn't think of any scenario where this would be re-used. So I decided to make it component-specific, but from the perspective of how intuitive it is, I can rename it :)